.github/workflows/set-rollouts.yaml

```diff
@@ -24,13 +24,14 @@ jobs:
           packageVersion: ${{ github.event.deployment.payload.packageVersion }}
 
           # Rollout information
-          deploymentDomain: "third-party-aggregator.decentraland.zone"
-          deploymentName: "_site"
+          deploymentPath: 'third-party-aggregator'
```

Comment:
We're planning to modify the site a bit and make it more user friendly. What do you think about using the `/linked-wearables` path here?

Reply:
sure, it's a great idea! updated :)